lib/vsc/utils/fancylogger.py

```diff
@@ -85,6 +85,13 @@
 import weakref
 from distutils.version import LooseVersion
 
+try:
```

Comment:
Easy to do, but why?  if the import succeeds, you can use the feature, otherwise it will be ignored. It does not have side-effects like probing for `mpi4py` can have...